modules/ROOT/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -31,9 +31,18 @@ include::6.5@sdk:shared:partial$acid-transactions.adoc[tag=requirements]
 
 Couchbase transactions require no additional components or services to be configured.
 Simply add the transactions library into your project.
-The latest version, as of October 2020, is 1.0.0.beta.1.
+The latest version is 1.0.0.
```

Comment:
Would be nice to have the release date or month/year.  We do this elsewhere as it helps people understand how far behind they are.

Reply:
yep why not